### DISCUSSION TITLE
EER frames for denoising training

### DISCUSSION THREAD

**Discussion by katciaz** [tbepler/topaz#107] [Q&A]
Hi,

I have successfully trained a model on my K3 data before, but now have Falcon4 eer frames on my hands. Can these be used as input for denoising? When I try to split them into even and odd (the vey first step in the tutorial), I get the following error:

`in parse(content)
    138     array = array[:header.nz*header.ny*header.nx]
    139     ## reshape the array
--> 140     array = np.reshape(array, (header.nz, header.ny, header.nx)) # , order='F')
    141     if header.nz == 1:
    142         array = array[0]

ValueError: cannot reshape array of size 0 into shape (425876394,8,2836809)
`

Best wishes,
Kat

**Comment by tbepler** [comment]
If you can align and sum these micrographs as normal, you can simply feed them into the pre-trained model. I recommend doing this as, unless the model performs badly on EER data for some reason (which I don't expect), there is no need to train dataset specific denoising models.
